index.html

```diff
@@ -50,127 +55,148 @@ <h1>I’m Mohammed Glad to see you!</h1>
+          <li>javaScript</li>
         </ul>
+        <button class="btn work-section__btn" type="submit">See Project</button>
       </div>
-      <p>
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required.
-      </p>
-      <ul class="programing-lamguages">
-        <li>html</li>
-        <li>css</li>
-        <li>javaScript</li>
-      </ul>
-      <button type="submit">See Project</button>
     </div>
-    <div class="card4 card-work">
-      <img src="./img/work4.svg" alt="" />
-      <h1>Multi-Post Stories</h1>
-      <div class="frame2">
-        <p>CANOPY</p>
-        <ul>
-          <li>Back End Dev</li>
-          <li>2015</li>
+
+    <div class="card4 card-work work-reversed-row">
+      <div class="card-work__img">
+        <img src="./img/work4.svg" alt="" />
+      </div>
+      <div class="card-work__info">
+        <h1>Multi-Post Stories</h1>
+        <div class="frame2">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="programing-lamguages">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
         </ul>
+        <button class="btn work-section__btn" type="submit">See Project</button>
       </div>
-      <p>
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required.
-      </p>
-      <ul class="programing-lamguages">
-        <li>html</li>
-        <li>css</li>
-        <li>javaScript</li>
-      </ul>
-      <button type="submit">See Project</button>
     </div>
   </section>
+
   <section class="about-section">
-    <div class="container">
-      <h1>About<br />Myself</h1>
-      <p>
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact
-        me.
-      </p>
-    </div>
-    <div class="contact">
-      <a href="">
-        <p>LET'S CONNECT</p>
-      </a>
-      <ul>
-        <li>
-          <a href="https://twitter.com/RebbouhElseddik"><img src="./img/twitter.svg" alt="" /></a>
-        </li>
-        <li>
-          <a href=""><img src="./img/Vector(1).svg" alt="" /></a>
-        </li>
-        <li>
-          <a href=""><img src="./img/Vector(2).svg" alt="" /></a>
-        </li>
-        <li>
-          <a href="https://github.com/MohammedElseddik">
-            <img src="./img/github.svg" alt="" /></a>
-        </li>
-        <li>
-          <a href=""><img src="./img/Vector(4).svg" alt="" /></a>
-        </li>
-      </ul>
-      <button type="submit">Get my resume</button>
+    <div class="about-me">
+      <div class="about-container">
+        <h1>About<br />Myself</h1>
```

Comment:
Kindly change the `h1` tag for `About Myself` to be any heading between `h2` and `h6`, for example: `h2`. since the `h1` tag should only exist once on your page.